privacyidea/lib/containerclass.py

```diff
@@ -69,13 +71,33 @@ def realms(self):
         return self._db_container.realms
 
     def set_realms(self, realms, add=False):
+        """
+        Set the realms of the container. If add is True, the realms will be added to the existing realms, otherwise the
+        existing realms will be removed.
+
+        :param realms: List of realm names
+        :param add: False if the existing realms shall be removed, True otherwise
+        :return: Dictionary of type {realm: success}, the entry 'deleted' indicates whether existing realms were deleted
```

Comment:
I would not use type, as it is an already occupied word and not quite fitting.
Suggestion: "Dictionary in the format: " or something like that